### PR TITLE
minor correction

### PR DESCRIPTION
**Pull request by joinimran**
Volume E was mentioned in the document but in actual case it should only b Volume.

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/6218

### CODE REVIEW

**Comment by joinimran** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

@e0i FYI please.

File: windows/security/information-protection/bitlocker/bitlocker-basic-deployment.md [hunk 1]
```diff
@@ -457,7 +457,7 @@ Checking BitLocker status with the control panel is the most common method used
...
  
-If a drive is pre-provisioned with BitLocker, a status of "Waiting for Activation" displays with a yellow exclamation icon on volume E. This status means that there was only a clear protector used when encrypting the volume. In this case, the volume is not in a protected state and needs to have a secure key added to the volume before the drive is fully protected. Administrators can use the control panel, manage-bde tool, or WMI APIs to add an appropriate key protector. Once complete, the control panel will update to reflect the new status.
+If a drive is pre-provisioned with BitLocker, a status of "Waiting for Activation" displays with a yellow exclamation icon on volume. This status means that there was only a clear protector used when encrypting the volume. In this case, the volume is not in a protected state and needs to have a secure key added to the volume before the drive is fully protected. Administrators can use the control panel, manage-bde tool, or WMI APIs to add an appropriate key protector. Once complete, the control panel will update to reflect the new status.
```
Comment (JohanFreelancer9) [inline windows/security/information-protection/bitlocker/bitlocker-basic-deployment.md:460]:
```suggestion
If a drive is pre-provisioned with BitLocker, a status of "Waiting for Activation" displays with a yellow exclamation icon on the volume. This status means that there was only a clear protector used when encrypting the volume. In this case, the volume is not in a protected state and needs to have a secure key added to the volume before the drive is fully protected. Administrators can use the control panel, manage-bde tool, or WMI APIs to add an appropriate key protector. Once complete, the control panel will update to reflect the new status.
```

Comment (JohanFreelancer9) [reply windows/security/information-protection/bitlocker/bitlocker-basic-deployment.md:460]: Please add "the" before volume.

**Comment by joinimran** [comment]
@e0i  - Ready to go live

**Pull request merged**